contract/external.go

```diff
@@ -0,0 +1,60 @@
+/**
+* This a temp file that mock the external APIs used in smart contract component.
+* It will be removed when blockchain infrastructure constructed, including account,
+* security, RPC, consensus and blockchain storage components.
+*
+* @file
+* @copyright defined in go-seele/LICENSE
+ */
+package contract
+
+import (
+	"github.com/seeleteam/go-seele/crypto"
+)
+
+const (
+	txTypeContractReg int = iota
```

Comment:
what's iota?

Reply:
auto increase value start with 0, this is golang syntax for enum definition.